docs/CHANGELOG.md

```diff
@@ -9,23 +9,79 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## DevilutionX 1.5.4
 
-### Bug Fixes
+### Features
 
 #### Multiplayer
 
+- Make invalid items unusable
+
+#### Controls
+
+- Better support for various gamepads
+- Gamepad: Improve navigation of the stash
+- Improve keymapper options
+
+#### Platforms
+
+- Added support for macOS 10.4 PowerPC
+- Added support for Windows XP
+- Android: Avoid game lagging on devices that can switch refresh rates
+- Correct networking support on big-endian systems
+- iOS: High refresh rate support
+- Xbox One/Series: Include translations
+
 #### Translations
 
-### Bugfixes
+- Improve rendering of Turkish texts
+- Update Italian translation
+- Update Russian translation
+- Update Ukrainian translation
+
+#### Stability / Performance / System
+
+- Fix crashes on device hot-plugging
+- Fixed game crash
```

Comment:
```suggestion
- Fix game crash
```

To keep consistent tense

Reply:
Also, IMO this should be more specific, for example "Fix game crash on startup"

Reply:
I don't think I can express the exact case in an easily understandable way.